### DISCUSSION TITLE
NVDA reads the first line of websites and emails with browse mode turned off

### DISCUSSION THREAD

**Discussion by joshknnd1982** [nvaccess/nvda#14941] [Ideas]
When a website, or email in outlook is first loaded, and browse mode is turned off, NVDA always reads the first line at the very top of the website or email. Please give us multiple options for automatic say all on page load, to either speak nothing and stay quiet, beep when page is fully loaded, or speak first line only.

**Comment by Brian1Gaff** [comment]
I have a feeling that the current situation was requested back very near the 
start of NVDA. I like hearing the first line. Under what condition might you 
not want to hear where you are?
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "joshknnd1982" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Saturday, May 20, 2023 4:48 PM
Subject: [nvaccess/nvda] NVDA reads the first line of websites and emails 
with browse mode turned off (Issue #14938)

**Comment by joshknnd1982** [comment]
I may not want to hear where I am and always have the first line read, and instead, have it beep, or stay silent, if I am more focused on another task, yet I want a small beep to let me know a document has been loaded.
